sdk/communication/azure-communication-rooms/azure/communication/rooms/_models/_models.py

```diff
@@ -0,0 +1,224 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+from datetime import datetime
+from typing import List, Optional, Union
+
+from azure.core.exceptions import DeserializationError
+from .._generated.models import (
+    RoomParticipant as RoomParticipantInternal,
+    CommunicationIdentifierModel,
+    CommunicationUserIdentifierModel,
+    RoomJoinPolicy,
+    RoleType
+)
+from .._generated import _serialization
+from .._generated.models import RoomModel
+from .._shared.models import (
+    CommunicationIdentifier,
+    CommunicationIdentifierKind,
+    CommunicationUserIdentifier,
+    UnknownIdentifier
+)
+
+
+class CommunicationRoom(_serialization.Model):
+    """The response object from rooms service.
+
+    :ivar id: Unique identifier of a room. This id is server generated.
+    :vartype id: str
+    :ivar created_on: The timestamp when the room was created at the server. The timestamp
+     is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
```

Comment:
These are all datetimes - so the string formatting shouldn't be in the docstring.